### PR TITLE
Create design landing section

### LINKED ISSUE #15: [Feature] (Design) Landing Section
### Description

Build the landing section on the Design Page

### Designs

image

### PR DESCRIPTION
**Pull request by gramliu**
# Description

Created the welcome blurb and headers. Added the software icons for Figma, Illustrator, Procreate, and After Effects.

Closes #15 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Screenshots

## Desktop
image

## Tablet
image

## Mobile
image
image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/icons/Icon/index.module.scss [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 %baseSvgIconContainer {
   width: 100%;
+  height: 100%;
```
Comment (hdavidethan) [inline src/icons/Icon/index.module.scss:6]: Does this break the footer? If you run the debugBbox, is the bbox still the same?

File: pages/design.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import AfterEffectsIcon from "../src/icons/AfterEffectsIcon";
+import FigmaIcon from "../src/icons/FigmaIcon";
+import IllustratorIcon from "../src/icons/IllustratorIcon";
+import ProcreateIcon from "../src/icons/ProcreateIcon";
+import styles from "../styles/Design.module.scss";
+
+const Design: NextPage = () => {
+  return (
+    <>
+      <Head>
```
Comment (hdavidethan) [inline pages/design.tsx]: Perhaps make a Layout for this?

**Review by hdavidethan — CHANGES_REQUESTED** [review]
What's the reason behind adding the svg files to public/icons?

File: src/icons/Icon/index.module.scss (hunk 1 above)
Comment (gramliu) [reply src/icons/Icon/index.module.scss:6]: Doesn't seem to break the footer and it still looks the same as far as I can tell
image

File: pages/design.tsx (hunk 2 above)
Comment (gramliu) [reply pages/design.tsx]: Yeah I'll change this to use `BaseLayout` once the other PR gets merged

**Review by hdavidethan — APPROVED** [review]

**Pull request merged**